install.py

```diff
@@ -879,12 +877,13 @@ def driver():
         help="Maximum number of fields that Legate will support",
     )
     parser.add_argument(
-        "--gasnet",
-        dest="gasnet",
-        action="store_true",
+        "--network",
+        dest="networks",
```

Comment:
We can feed multiple networks to Realm, but I am not advertising this currently, since this feature is not fully working on the Realm side.